Userland/Utilities/cal.cpp

```diff
@@ -35,12 +35,26 @@ static ErrorOr<StringView> month_name(int month)
     return AK::long_month_names.at(month_index);
 }
 
-static ErrorOr<Vector<String>> month_lines_to_print(int month, int year)
+enum class Header {
+    MonthAndYear,
+    Month,
+};
+
+static ErrorOr<Vector<String>> month_lines_to_print(Header header_mode, int month, int year)
 {
     Vector<String> lines;
 
-    // FIXME: Both the month name and month header text should be provided by a locale
```

Comment:
The FIXME about locales is lost in commit `cal: Only show year next to each month in one-month-mode`

Reply:
Done